README.md

```diff
@@ -1,3 +1,14 @@
 # Darling Docs
 
 This repository stores the contents of https://docs.darlinghq.org
+
+Building
```

Comment:
This is a nitpick, but I think a title along the lines of `Generating A Local Copy` would be better then `Building`.

Reply:
> 

Mostly I proposed this change because I couldn't figure out which command to use, and then where to get it from (not in Ubuntu 20.04 LTS repos), so I thought it might lower the barrier to entry for contributions to have this in the readme. If you don't like the changes, that's fine and I'm happy to close the merge request :)

Reply:
I'm happy to make that change, but let me know if you're interested in the PR first.